src/Microsoft.DotNet.XHarness.Android/AdbRunner.cs

```diff
@@ -387,7 +387,7 @@ public List<string> PullFiles(string devicePath, string localPath)
                         retriesLeft = 30; // Max 5 minutes (30 attempts * 10 second waits)
                         while (retriesLeft-- > 0 && shellResult.StandardError.Contains("device offline", StringComparison.OrdinalIgnoreCase))
                         {
-                            _log.LogWarning($"Device '{deviceSerial}' is offline; retrying up to one minute.");
+                            _log.LogDebug($"Device '{deviceSerial}' is offline; retrying up to one minute.");

```

Comment:
There might be another way around this and keep it as `Warning` to not worsen the experience for all the other flows which look for devices.

You can set the debugging level for that specific command to `Error` by overriding the value in the arguments object.

Maybe that would be better?

Reply:
great idea, thanks!

Reply:
I cannot overload it that way because logger with some level of verbosity created very early in `Main`. But I may set required verbosity as a parameter

Reply:
You should be able to do following in `AndroidGetDeviceCommand`:
```
        private readonly AndroidGetDeviceCommandArguments _arguments = new()
        {
            Verbosity = LogLevel.Error
        };
```